PULL_REQUEST_TEMPLATE.md

```diff
@@ -20,3 +20,4 @@ _Omschrijving van de benodigde (technische) stappen om een testbare situatie te
 - [ ] Acceptatie criteria van de story gecheckt
 - [ ] README bijgewerkt (indien van toepassing)
 - [ ] Confluence bijgewerkt (indien van toepassing)
+- [ ] Doctrine migraties zijn getest op de accept db
```

Comment:
Dit is een checklist voordat het PR is gemerged, het testen van deze checkbox kan pas na de merge en hoort daarom hier niet thuis.

Reply:
Nee, want je moet je migration testen op de accept db die je aan jeroen vraagt VOORDAT je merged ;-)

Reply:
Volgens mij ligt hier een ander probleem aan de grondslag, namelijk hoe kan het dat de migratie lokaal wel werkt en niet accept?

Reply:
Dat weet je alleen door de migratie uit te voeren op de accept db. Dit komt voornamelijk door FK's die op accept wel bestaan en op fixture data niet bijv.

Reply:
Ik heb de tekst aangepast naar "Doctrine migraties zijn lokaal getest op de accept db" om verwarring te voorkomen.